src/commitizen/configLoader.js

```diff
@@ -1,10 +1,13 @@
-import { loader } from '../configLoader';
+import { cosmiconfig } from "cosmiconfig";
 
 export { load };
 
-// Configuration sources in priority order.
-var configs = ['.czrc', '.cz.json', 'package.json'];
+async function load(config, cwd) {
+  const explorer = cosmiconfig("commitizen", {
+    packageProp: "config.commitizen",
```

Comment:
Look for the `config.commitizen` property in package.json. Does it also need to look for a commitizen property at the root?